toast-tk-runtime/src/main/java/io/toast/tk/runtime/utils/EncryptHelper.java

```diff
@@ -0,0 +1,28 @@
+package io.toast.tk.runtime.utils;
+
+public class EncryptHelper {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)